pkg/operator/deploy/deploy.go

```diff
@@ -138,6 +138,15 @@ func (o *operator) resources() ([]kruntime.Object, error) {
 		domain += "." + o.env.Domain()
 	}
 
+	ingressIP := o.oc.Properties.IngressProfiles[0].IP
+	if o.oc.Properties.IngressProfiles != nil && len(o.oc.Properties.IngressProfiles) > 1 {
+		for _, p := range o.oc.Properties.IngressProfiles {
+			if p.Name == "router-default" {
```

Comment:
I think the name is `default` not `router-default`

```json
    "ingressProfiles": [
      {
        "ip": "1.2.3.4",
        "name": "default",
        "visibility": "Public"
      }
    ],
```

Reply:
Fixed, Thanks!!